app/models.py

```diff
@@ -2324,3 +2324,24 @@ class FeatureToggle(db.Model):
 
     def serialize(self):
         return {"name": self.name, "is_enabled": self.is_enabled, "display_html": self.display_html}
+
+
+class FailedLoginCountByIP(db.Model):
+    """
+    This table is used to store IP addresses with the count of failed logins.
```

Comment:
just modify this comment to say "This table is used to store failed logins", as the count is no longer stored